runtime/lua/vim/lsp/handlers.lua

```diff
@@ -185,7 +185,7 @@ local function response_to_list(map_result, entity)
           title = 'Language Server';
           items = map_result(result, ctx.bufnr);
         })
-        api.nvim_command("copen")
+        api.nvim_command("botright copen")
```

Comment:
Shouldn't we do the same for the lopen call further above to have the two have consistent behavior?

Reply:
Actually, given that the location list is bound to the window I guess the difference makes sense.